cmake/BLTMacros.cmake

```diff
@@ -596,17 +596,18 @@ endmacro(blt_add_executable)
 
 
 ##------------------------------------------------------------------------------
-## blt_add_test( NAME           [name]
-##               COMMAND        [command] 
-##               NUM_MPI_TASKS  [n]
-##               CONFIGURATIONS [config1 [config2...]])
+## blt_add_test( NAME            [name]
+##               COMMAND         [command] 
+##               NUM_MPI_TASKS   [n]
+##               NUM_OMP_THREADS [n]
```

Comment:
Would ``NUM_CPU_THREADS`` be better?

Reply:
I am torn on this.  On one hand OMP is specific, it avoids future conflicts with other threading libraries (maybe not a problem but i like being specific).  

I have spoken to several people (and plan on a few more) and the consensus so far is what I have.